### ISSUE TITLE
Xlarig JSON causes errors in the log

### ISSUE THREAD

**Issue by Boulder08** [RainbowMiner/RainbowMiner#2833]
RBM started benchmarking Xlarig with Panthera, and a lot of these errors popped up in the console. Benchmarking seemed to work fine though.

PS>TerminatingError(ConvertFrom-Json): "Conversion from JSON failed with error: Unexpected end when reading token. Path 'hashrate.threads[0]'."

**Comment by RainbowMiner** [comment]
Yes, that is a known issue with that miner. RainbowMiner already does some magic to read the broken API, but the call to the API itself causes that error. I will push another commit that should get rid of this message.

**Comment by RainbowMiner** [comment]
Fixed - the API call that produced the log noise was changed. Closing as completed - thanks for the report!
